### PR TITLE
Updating documentation and demos to use homeAccountId and email properties of account

### PR DESCRIPTION
**Pull request by chbhatt**
Updating documentation and demos to use homeAccountId and email properties of account

### CODE REVIEW

File: docs/quick-authentication-reference.md [hunk 1]
```diff
@@ -148,8 +148,10 @@ This data type contains following values.
 | `surname` | User's surname |
 | `givenName` | User's given name |
 | `username` | User's email or phone number |
+| `email` | User's email |
```
Comment (christianfo) [inline docs/quick-authentication-reference.md:151]: you might want to say "User's email address" to be more specific.

File: docs/quick-authentication-reference.md [hunk 2]
```diff
@@ -162,8 +164,10 @@ This data type contains all the fields of [AccountInfo](#data-type-accountinfo)
 | `surname` | the user's surname |
 | `givenName` | the user's given name |
 | `username` | User's email or phone number |
+| `email` | User's email |
```
Comment (christianfo) [inline docs/quick-authentication-reference.md:167]: Same comment as above

File: docs/quick-authentication-how-to.md [hunk 3]
```diff
@@ -169,8 +169,10 @@ The [SignInAccountInfo](./quick-authentication-reference.md#data-type-signinacco
 - `surname` - the user's surname
 - `givenName` - the user's given name
 - `username` - the user's email address or phone number
+- `email` - the user's email
```
Comment (christianfo) [inline docs/quick-authentication-how-to.md:172]: Suggestion: "User's email address" to be more explicity

**Review by christian-fortini — APPROVED** [review]
See a few comments I left under my other identity.

File: docs/quick-authentication-reference.md (hunk 1 above)
Comment (chbhatt) [reply docs/quick-authentication-reference.md]: Changed

**Pull request merged**